src/bios_handler.cpp

```diff
@@ -460,4 +463,93 @@ void IbmBiosHandler::processClearNvram()
     }
     logging::logMessage("Invalid type recieved for clear NVRAM from VPD.");
 }
+
+void IbmBiosHandler::saveKeepAndClearToVpd(const std::string& i_KeepAndClearVal)
+{
+    if (i_KeepAndClearVal.empty())
+    {
+        logging::logMessage(
+            "Empty value received for keep and clear from BIOS. Skip updating to VPD.");
+        return;
+    }
+
+    // Read required keyword from DBus as we need to set only a Bit.
+    auto l_kwdValueVariant = dbusUtility::readDbusProperty(
+        constants::pimServiceName, constants::systemVpdInvPath,
+        constants::utilInf, constants::kwdClearNVRAM_CreateLPAR);
```

Comment:
should it be kwdKeepAndClear ?

Reply:
Yes but if you check my comment in const.hpp file. 
Both are pointing to same kwd and I was doubtful about the having duplicate in const. So till that gets approved I kept this as it is.
As decided now, we will have duplicate entry in const in some rare situations like this.